app/lib/widgets/cards/productCard/shared/product_infos.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flex_ui/flex_ui.dart';
+import 'package:flex_ui/utils/extension.dart';
+import 'package:flex_ui/widgets/cards/productCard/shared/product_notation.dart';
+import 'package:flutter/material.dart';
+
+class ProductInfos extends StatelessWidget {
```

Comment:
Nitpicky, but can we rename to `ProductInfo` (singular "information")

Reply:
Done :)